hackathon_site/dashboard/frontend/src/components/cart/CartErrorBox/index.tsx

```diff
@@ -19,6 +27,18 @@ const CartErrorBox = () => {
                         title="Unable to submit your order because:"
                     />
                 )}
+                {(teamSizeTooLarge || teamSizeTooSmall) && (
+                    <Alert severity="warning" style={{ margin: "15px 0px" }}>
+                        <AlertTitle>{`Team Size Too ${errorTitle}!`}</AlertTitle>
+                        {`There are too ${errorMessage} people on your team to place an
+                        order. We only allow teams between ${minTeamSize} to ${maxTeamSize} 
```

Comment:
Sorry me english is bad, this sentence should be "We only allow teams between {min} to {max} **people** to checkout items"